po/de.po

```diff
@@ -518,41 +507,41 @@ msgstr "Sperren der Geschichts-Datei dauerte zu lang (%.3f Sekunden)."
 #: src/history.cpp:887
 #, c-format
 msgid "Error when renaming history file: %s"
-msgstr ""
+msgstr "Fehler beim Umbenennen der History-Datei: %s"
```

Comment:
"Historiendatei"?

Reply:
Or "Verlauf"? Bash uses "Befehlswiederholungsliste" which is descriptive but frankly awful.

Reply:
Wtfudge, yeah that's terrible. "Verlauf" would be alright.